.github/workflows/ng-update.yml

```
@@ -0,0 +1,13 @@
+name: "Update Angular Action"
+on: # when the action should run. Can also be a CRON or in response to external events. see https://git.io/JeBz1
+  schedule:
+    - cron: '0 5 1 * *'
```

```suggestion
    - cron: '0 5 1 * *'
  workflow_dispatch:
```

If you add this we can manually trigger the workflow without waiting for the right time to roll around.

---

Good catch – thanks!